src/scope.js

```diff
@@ -400,19 +422,19 @@ class Scope {
     }
 
     __isClosed() {
-        return this.__left === null;
+        return this.__left === null || this.__left === undefined;
```

Comment:
As I say, I have no idea what significance that has, but it's probably worth pointing out